cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/ComputeEngineCredentials.java

```diff
@@ -156,7 +128,7 @@ public void checkServerTrusted(X509Certificate[] arg0, String arg1) {
     }
   }
 
-  private AccessToken fetchAccessToken(String endPoint) throws IOException {
+  private AccessToken refreshAccessTokenRemotely(String endPoint) throws IOException {
```

Comment:
nit: Change to `getAccessTokenRemotely`

Reply:
nit: Similarly let's change refreshAccessTokenLocally to `getAccessTokenLocally`